src/Hl7.FhirPath/FhirPath/Expressions/EvaluatorVisitor.cs

```diff
@@ -16,33 +16,41 @@ namespace Hl7.FhirPath.Expressions
 {
     internal class EvaluatorVisitor : FP.ExpressionVisitor<Invokee>
     {
-        public override Invokee VisitConstant(FP.ConstantExpression expression, SymbolTable scope)
+        public SymbolTable Symbols { get; }
```

Comment:
Is it necessary that this property `Symbols` is public?

Reply:
Discussed together: this property is part of a internal class, so it is not very important.